src/blocks/utils/GifViewer.js

```diff
@@ -1,10 +1,8 @@
 import React, { useState, useEffect } from "react";
-import { Gif } from "@giphy/react-components"
```

Comment:
is this file unused now? prefer to not duplicate functionality in two different places